### ISSUE TITLE
LSP not parsing .all() syntax

### ISSUE THREAD

**Issue by mateenkasim** [authzed/spicedb-vscode#20]
### What platforms are affected?

macos

### What architectures are affected?

arm64

### What SpiceDB version are you using?

Latest version of the VSCode extension, 0.0.7: https://marketplace.visualstudio.com/items?itemName=authzed.spicedb-vscode

### Steps to Reproduce

Just following https://authzed.com/docs/spicedb/concepts/schema#all-intersection-arrow:
```
definition file {
  relation project: project

  permission access = project.all(access)
}
```

### Expected Result

Validates properly

### Actual Result

<img width="793" alt="image" src="https://github.com/user-attachments/assets/f88c5560-984e-48df-acd8-d2b28a371e6d">

**Comment by tstirrat15** [comment]
I'm at an offsite right now but I'll take a look at this soon.

**Comment by tstirrat15** [comment]
It should be sufficient to update the version in [download-wasm](https://github.com/authzed/spicedb-vscode/blob/main/download-wasm.sh) and regenerate.

**Comment by tstirrat15** [comment]
@mateenkasim I've cut https://github.com/authzed/spicedb-vscode/releases/tag/v1.0.0. Does that solve your problem?

**Comment by mateenkasim** [comment]
Unfortunately it did not :/

Also fyi, I got this warning when updating:
<img width="309" alt="image" src="https://github.com/user-attachments/assets/09295461-dadd-4347-8c9f-784a1782e1ce">

**Comment by tstirrat15** [comment]
@mateenkasim thank you for checking! I'll look into that on monday.

**Comment by tstirrat15** [comment]
@mateenkasim I realized that the issue you're running into is because [the extension uses the SpiceDB binary for its LSP](https://github.com/authzed/spicedb-vscode/blob/main/src/extension.ts#L259-L270). Upgrading your system's SpiceDB binary should resolve your issue.

I also figured out that we were able to simplify our publishing steps, so I'm going to cut a new release and see if that helps with the package signature problem.
